src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+@import '../utils/mixins';
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
```

Comment:
Be sure to include a `margin-right: 0;` for the last star to prevent an unnecessary margin on the right side. You can use the `:last-child` selector to target the last element.